packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -426,7 +446,9 @@ class _HelperErrorState extends State<_HelperError> with SingleTickerProviderSta
   Widget build(BuildContext context) {
     if (_controller.isDismissed) {
       _error = null;
-      if (widget.helperText != null) {
+      if (widget.errorWidget != null) {
```

Comment:
Why is this new case needed here?

Reply:
I thinks It's now not needed I am refactoring the code and you can see the fixes on next commit.